opus/application/run_coverage.sh

```
@@ -1,4 +1,6 @@
-coverage run ../../lib/opus_support.py
+coverage erase
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)